### ISSUE TITLE
SuiteCRM records are not assigned with the sync-user's Security Group

### ISSUE THREAD

**Issue by fosullivan-usam** [Myddleware/myddleware#314]
When Myddleware creates records (of any kind, e.g., Accounts) in an instance of SuiteCRM, it DOES NOT assign the Security Group of the sync user's account to the record.  Whereas, if you log in to the SuiteCRM instance through a browser using the same username and password, when you create a record, it DOES assign the user's Security Group to the record.

**Comment by Myddleware** [comment]
Hi @fosullivan-usam ,
I have done the same test than you and the contact created by Myddleware in SuiteCRM has been attached to the user group (used in Myddleware connector) : 
![image](https://user-images.githubusercontent.com/11330301/83428571-ec6b5a80-a432-11ea-9088-a1431b616e6e.png)

For information, groups are added only when Myddleware creates the record, not when Myddleware modifies it.

Could you verify if you have checked this in Security Groups Management configuration (in SuiteCRM) : 
![image](https://user-images.githubusercontent.com/11330301/83428486-c3e36080-a432-11ea-819f-fd4eb6b19c6d.png)

Please let me know if it helps.

**Comment by fosullivan-usam** [comment]
Thank you for the quick turn-around.  I had both issues you mentioned:
1. I did not have 'Inherit from Created By User' checked (only Inherit from Assigned To User - I wonder why that wasn't enough).
2. I had synchronized most of the records before checking the box above, and so they didn't update even if they were modified.

My solution was to check the 'Inherit from Created By User' box, and Mass Update all the existing records with the correct Security Group assignment.
